app/views/applications/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Adoption Application</h1>
+
+<%= form_with url: "/applications/:id", method: :post, local: true, id: "new_application" do |form| %>
```

Comment:
hey i think this should be `/applications` because of the `method: :post` it will still make it to the correct controller action. I saw the create is redirecting to the show so that is fine.

Reply:
Thank you for pointing this out - now updated!